composer.json

```diff
@@ -12,6 +12,8 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.0",
+        "ext-mysqlnd": "*",
+        "ext-pdo_mysql": "*",
```

Comment:
I may still be wrong but don't we really need `ext-pdo` and `ext-pdo_mysql`? not looking from program side here but from how PHP has distributed its functionalities?

Reply:
Composer checks if a php extension is installed and active. Similar to checking "php -m".
PHP extension naming is the same on all OS. PHP slices its functionalities always in the same list of extensions (as long as we look at a specific php version). There is no difference between 7.0., 7.1 and 7.2 for the PR related extensions and nothing announced for further releases.

Do not be confused be OS packaging. The above 3 extensions are bundled as one debian package on current raspbian. Other OS may create other bundles, but this is not relevant when we define dependencies.